bruinen/src/bruinen/langchain/parsers/sql_parser.py

```diff
@@ -97,13 +78,9 @@ def map_sql_types(type_var):
 
         metadata = MetaData()
         metadata.reflect(bind=engine)
-        sql_table = Table(
-            input_class_name.lower(), metadata, autoload=True, autoload_with=engine
-        )
+        sql_table = Table(input_class_name.lower(), metadata, autoload=True, autoload_with=engine)
 
-        template_table = "\nOnly use the following tables:\n" + str(
-            CreateTable(sql_table).compile(engine)
-        )
+        template_table = "\nOnly use the following tables:\n" + str(CreateTable(sql_table).compile(engine))
         # If the table isn't too large, would be better to also include example data
         template_start = """Given an input question, first create a syntactically correct {dialect} query to run, then look at the results of the query and return the answer. Use the following format:
```

Comment:
Lets experiment with different prompts to see if we can get even better results?

Lets make a backlog task for this (low priority), fun thing to work on in any downtime